src/inc/msquic_posix.h

```diff
@@ -218,6 +218,10 @@ extern "C" {
 // IP Address Abstraction Helpers
 //
 
+// RFC 6335
+#define QUIC_ADDR_EPHEMERAL_PORT_MIN 49152
+#define QUIC_ADDR_EPHEMERAL_PORT_MAX 65535
```

Comment:
At least on Windows, I think these ranges are customizable. Are they on Linux as well?

Reply:
Yes, it is customizable on both OSes, but these ranges are the recommended defaults via RFC. I think Linux uses a larger default range, however.
Could query for these values instead, would that be preferable?

Reply:
Let's start with what you have. But I see no reason to define this in 3 different platform-specific places, considering they are all the same. Let's move this to quicdef.h or platform.h.